aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,137 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import PropTypes from 'prop-types';
+import {
+  AnnounceContext,
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+const Announcer = ({ announce, message, mode, role }) => {
+  const theme = useContext(ThemeContext);
+  announce(message, mode);
+  return (
+    <Text role={role} aria-live={mode} {...theme?.feedback?.success}>
+      {message}
+    </Text>
+  );
+};
+
+const AnnounceContextComponent = props => (
+  <AnnounceContext.Consumer>
+    {announce => <Announcer announce={announce} {...props} />}
+  </AnnounceContext.Consumer>
+);
+
+export const Feedback = ({
+  children,
+  layerProps,
+  messages,
+  modal,
+  onClose,
+  onSubmit,
+  show,
+  title,
+}) => {
+  const theme = useContext(ThemeContext);
+  const breakpoint = useContext(ResponsiveContext);
+  // tracks if feedback has successfully been submitted
+  const [successfulSubmit, setSuccessfulSubmit] = useState(false);
+
+  useEffect(() => {
+    if (show) setSuccessfulSubmit(false);
+  }, [show]);
+
+  let footerContent;
+  if (!successfulSubmit) {
+    footerContent = (
+      <Box {...theme?.feedback?.footer}>
+        <Button onClick={onClose} label={messages?.cancel || 'Cancel'} />
+        <Button label={messages?.submit || 'Submit'} primary type="submit" />
+      </Box>
+    );
+  } else
+    footerContent = (
+      <Box {...theme?.feedback?.footer}>
+        <AnnounceContextComponent
+          mode="assertive"
+          role="alert"
+          message={messages?.successful || 'Thanks'}
```

Comment:
```suggestion
          message={messages?.successful || 'Thank you!'}
```
Should this align with the text that appears visually when you submit?